extensions/amp-accordion/1.0/accordion.js

```diff
@@ -397,16 +415,47 @@ export function AccordionContent({
 }) {
   const ref = useRef(null);
   const hasMountedRef = useRef(false);
-  const {contentId, headerId, expanded, animate, setContentId} = useContext(
-    SectionContext
-  );
+  const {
+    contentId,
+    headerId,
+    expanded,
+    animate,
+    setContentId,
+    expandHandler,
+    experimentDisplayLocking,
+  } = useContext(SectionContext);
   const classes = useStyles();
+  const [contentVisibility, setContentVisibility] = useState(false);
+  const useDisplayLocking = contentVisibility && experimentDisplayLocking;
```

Comment:
Can't call vars with `use` prefix - those are reserved for hooks only in React.

Reply:
Updated to `enableDisplayLocking`